File: src/main/java/pl/arturczopek/mycoach/repository/SeriesRepository.java
package pl.arturczopek.mycoach.repository;

import org.springframework.data.repository.CrudRepository;
import pl.arturczopek.mycoach.model.database.Series;

/**
 * @Author Artur Czopek
 * @Date 09-10-2016
 */

public interface SeriesRepository extends CrudRepository<Series, Long> {

    void deleteByExerciseSessionId(long exerciseSessionId);
}

File: src/main/kotlin/pl/arturczopek/mycoach/repository/SeriesRepository.kt
package pl.arturczopek.mycoach.repository

import org.springframework.data.repository.CrudRepository
import pl.arturczopek.mycoach.model.database.Series

/**
 * @Author Artur Czopek
 * @Date 09-10-2016
 */

interface SeriesRepository : CrudRepository<Series, Long>
